pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -31,6 +31,7 @@ import (
 
 var podCondSet = apis.NewLivingConditionSet(
 	PodAutoscalerConditionActive,
+	MetricConditionReady,
```

Comment:
Do we really want to update PA status with metric status?
In any case that was not covered by the issue, so this might need more thinking.
I'd restrict the PR solely to the level of adding and setting the status on the metric object.

Reply:
I see. It makes sense to make this PR strictly about updating the metric status. I'll make the change.